packages/cli/api-importers/v2-importer-commons/src/converters/schema/SchemaConverter.ts

```diff
@@ -173,11 +193,209 @@ export class SchemaConverter extends AbstractConverter<AbstractConverterContext<
         };
     }
 
-    public convertDeclaredTypeName({ context }: { context: AbstractConverterContext<object> }): DeclaredTypeName {
+    public convertDeclaredTypeName({
+        context
+    }: {
+        context: AbstractConverterContext<object>;
+    }): FernIr.DeclaredTypeName {
         return {
             typeId: this.id,
             fernFilepath: context.createFernFilepath(),
             name: context.casingsGenerator.generateName(this.id)
         };
     }
+
+    private async createTypeDeclarationFromFernType({
+        fernType,
+        context,
+        errorCollector
+    }: {
+        fernType: string;
+        context: AbstractConverterContext<object>;
+        errorCollector: ErrorCollector;
+    }): Promise<FernIr.TypeDeclaration | undefined> {
```

Comment:
Discussed offline - it'd be great if the `errorCollector` was part of the `context` so we didn't have to manage it alongside the other parameters.